src/System.Runtime.Caching/src/System/Runtime/Caching/Dbg.cs

```diff
@@ -523,8 +529,8 @@ private class Tag {
             }
             else {
                 if (s_includeThreadPrefix) {
-                    idThread = NativeMethods.GetCurrentThreadId();
-                    idProcess = NativeMethods.GetCurrentProcessId();
+                    idThread = Thread.CurrentThread.ManagedThreadId;
+                    idProcess = Process.GetCurrentProcess().Id;
```

Comment:
The Process returned from GetCurrentProcess should be disposed of, e.g.
```C#
using (Process p = Process.GetCurrentProcess())
{
    idProcess = p.Id;
}
```

Reply:
Thanks @stephentoub!